recipes/fitsverify/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "fitsverify" %}
+{% set version = "4.20" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://heasarc.gsfc.nasa.gov/docs/software/ftools/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e4f8d34414ad441414386c81f2d241c800025cd1ad21c269d061484e617cef8
+
+build:
+  number: 0
```

If you don't want to support windows (and it's not a `noarch` package) then this is the right thing to do:
```suggestion
  number: 0
  skip: true  # [win]
```